src/jobs/controllers/jobController.ts

```diff
@@ -79,13 +89,16 @@ export class JobController {
 
   public updateStatus: TypedRequestHandlers['PUT /jobs/{jobId}/status'] = async (req, res, next) => {
     try {
-      await this.manager.updateStatus(req.params.jobId, req.body.jobOperationStatus);
+      await this.manager.updateStatus(req.params.jobId, req.body.status);
+
       return res.status(httpStatus.OK).json({ code: successMessages.jobModifiedSuccessfully });
     } catch (err) {
       if (err instanceof JobNotFoundError) {
         (err as HttpError).status = httpStatus.NOT_FOUND;
+        this.logger.error(`Job status update request failed: job with provided ID not found`, err);
```

Comment:
no need for this log as it will be logged anyway. only log if you have relevant information to add.

Reply:
ok, removed